doc/Language/performance.pod6

```diff
@@ -159,58 +176,66 @@ and/or L<the matching video|https://www.youtube.com/watch?v=JpqnNCx7wVY&list=PLR
 
 =head2 Use existing high performance code
 
-Is there an existing high (enough) performance implementation of what you're trying to speed up / slim down?
+Is there an existing high (enough) performance implementation of what you're
+trying to speed up / slim down?
 
-There are a lot of C libs out there.
-L<NativeCall|/language/nativecall> makes it easy to create wrappers for C libs (there's experimental
-support for C++ libs too) such as L<Gumbo|https://github.com/Skarsnik/perl6-gumbo>.
-(Data marshalling and call handling is somewhat poorly optimized at the time of writing this but for many
-applications that won't matter.)
+There are a lot of C libs out there. L<NativeCall|/language/nativecall> makes it
+easy to create wrappers for C libs (there's experimental support for C++ libs
+too) such as L<Gumbo|https://github.com/Skarsnik/perl6-gumbo>. (Data marshalling
+and call handling is somewhat poorly optimized at the time of writing this but
+for many applications that won't matter.)
 
-Perl 5's compiler can be treated as a C lib. Mix in Perl 6 types, the L<MOP|/language/mop>, and some hairy
-programming that someone else has done for you, and the upshot is that you can conveniently
-L<use Perl 5 modules in Perl 6|http://stackoverflow.com/a/27206428/1077672>.
+Perl 5's compiler can be treated as a C lib. Mix in Perl 6 types, the
+L<MOP|/language/mop>, and some hairy programming that someone else has done for
+you, and the upshot is that you can conveniently L<use Perl 5 modules in Perl
+6|http://stackoverflow.com/a/27206428/1077672>.
```

Comment:
We use `U+00A0 NO-BREAK SPACE` between `Perl` and `6` (or `5`), so that when viewing, the number doesn't accidentally get wrapped on a new line. Here, a newline got added in-between.